doc/release-notes/5.1.1-release-notes.md

```diff
@@ -0,0 +1,59 @@
+# Dataverse 5.1.1
+
+This minor release adds important optimizations to installations running on AWS S3. It is recommended that 5.1.1 be used in production instead of 5.1.
+
+## Release Highlights
+
+### Connection Pool Size Configuration Option, Connection Optimizations
+
+This release adds optimizations around closing streams and channels that may hold S3 http connections open and exhaust the connection pool. In parallel, this release adds the ability to increase the size of the connection pool, so a larger pool can be configured if needed.
+
+## Major Use Cases
+
+Newly-supported use cases in this release include:
+
+- Administrators of installations using S3 will be able to define the connection pool size, allowing better resource scaling for larger installations (Issue #7309, PR #7313)
+
+## Notes for Dataverse Installation Administrators
+
+### 5.1.1 vs. 5.1 for Production Use
+
+As mentioned above, we encourage 5.1.1 instead of 5.1 for production use.
+
+### New JVM Option for Connection Pool Size
+
+Larger installations may want to increase the number of open S3 connections allowed (default is 256). For example, to set the value to 4096:
+
+``./asadmin create-jvm-options "-Ddataverse.files.<id>.connection-pool-size=4096"`
+
+The JVM Options section of the [Configuration Guide](http://guides.dataverse.org/en/5.1.1/installation/config/) has more information.
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.1.1 Milestone](https://github.com/IQSS/dataverse/milestone/91?closed=1) in Github.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](http://guides.dataverse.org/en/5.1.1/installation/)
+
+## Upgrade Instructions
+
+0. These instructions assume that you've already successfully upgraded from Dataverse 4.x to  Dataverse 5 following the instructions in the [Dataverse 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0).
```

Comment:
This sort of implies that you can skip 5.1 (going straight from 5.0 to 5.1.1) but it looks like there's a metadata block and Solr change in 5.1. I guess we have a couple options:

- Tell people to upgrade to 5.1 first (more straightforward, I think).
- Copy to the 5.1.1 upgrade instructions information about biomedical.tsv and Solr change (more confusing, I think).

Reply:
Yeah  - people are used to having to install all intermediate versions, so I'd vote for just adding a reminder.

Reply:
Thanks @pdurbin (and @kcondon too) for the suggestion, updated in https://github.com/IQSS/dataverse/pull/7319/commits/155dfb6030f4b8bd85ea17e9c8b98cc303038968